acceptancetests/assess_caas_bootstrap.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+""" Test caas k8s cluster bootstrap
+"""
+
+from __future__ import print_function
+
+import argparse
+import logging
+import sys
+import os
+
+from deploy_stack import (
+    BootstrapManager,
+)
+from utility import (
+    add_basic_testing_arguments,
+    configure_logging,
+)
+
+from jujucharm import (
+    local_charm_path
+)
+
+
+__metaclass__ = type
+
+
+log = logging.getLogger("assess_caas_bootstrap")
+
+
+def assess_caas_bootstrap(client):
+    # Deploy k8s bundle to spin up k8s cluster
+    repository_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'repository')
+    bundle = local_charm_path(charm='bundles-kubernetes-core-lxd.yaml',
+                              repository=repository_dir, juju_ver=client.version)
+    client.deploy_bundle(bundle, static_bundle=True)
+    # Wait for the deployment to finish.
+    client.wait_for_started()
+
+    # ensure kube credentials
+    client.scp(_from='kubernetes-master/0:config', _to='~/.kube/config')
+
+    # TODO:
```

Comment:
Can you expand on what these steps will entail? This would help shape what k8sclient and CaasClient might look like.